### DISCUSSION TITLE
v2.0.26

### DISCUSSION THREAD

**Discussion by DTTerastar** [ESPresense/ESPresense#277] [Announcements]
## Changes

- Attempts at more stability @DTTerastar (#276)

<hr /><em>This discussion was created from the release <a href='https://github.com/ESPresense/ESPresense/releases/tag/v2.0.26'>v2.0.26</a>.</em>

**Comment by RutgerDiehard** [comment]
After installing 2.0.26, none of my 8 Atom Lites stay up longer than about 20 seconds. :-( 

All releases before that were seemingly stable.

**Reply by McFranky** [reply]
I have to WRoom32 with the same problems. Max 15sec online on 2.0.26

**Reply by erwinSmith757** [reply]
I use a D1 mini... and same problem, tried circumventing it with a script restarting every 5min.... but no luck

but the progress is impressive .... cant wait for a stable release

**Reply by HeavyHawkZero** [reply]
Same here with 3 WROOM32s

**Reply by feudas** [reply]
here also. esp32 fresh install. restart loop from watchdog

**Reply by gabroid** [reply]
Hi, first of all , I would you know that in my opinion this project is very awesome, i would like to thank you for your effort! 
I also want to inform you that i try [SODIAL GPS Mini Tag Smart Tracker Bluetooth](https://www.amazon.it/gp/product/B07DXQKD9N/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&psc=1) and it works fine (attention battery is not included and comes from China)

About this topic, same here, i got 9 new [ESP-WROOM-32 ESP-32 ESP-32S](https://www.amazon.it/gp/product/B08CCYWZN3/ref=ppx_yo_dt_b_asin_title_o01_s00?ie=UTF8&psc=1)  installed yesterday, and all report this kind of error:

```
E (8000) task_wdt: Task watchdog got triggered. The following tasks did not reset the watchdog in time:
E (8000) task_wdt:  - IDLE0 (CPU 0)
E (8000) task_wdt: Tasks currently running:
E (8000) task_wdt: CPU 0: btController
E (8000) task_wdt: CPU 1: IDLE1
E (8000) task_wdt: Aborting.
abort() was called at PC 0x4018b210 on core 0

ELF file SHA256: 0000000000000000

Backtrace: 0x4008f658:0x3ffbfee0 0x4008f8d5:0x3ffbff00 0x4018b210:0x3ffbff20 0x4008dd01:0x3ffbff40 0x4000bfed:0x3ffb5fc0 0x40091a59:0x3ffb5fd0 0x40081c7b:0x3ffb5ff0 0x4008d95a:0x3ffb6010 0x40191839:0x3ffb6030 0x40191e29:0x3ffb6050 0x400908e6:0x3ffb6080

Rebooting...
ets Jul 29 2019 12:21:46

rst:0xc (SW_CPU_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0018,len:4
load:0x3fff001c,len:1044
load:0x40078000,len:8896
load:0x40080400,len:5828
entry 0x400806ac
```


Also only for 1 binary_sensor.espresense_$(roomname)_motion is created, very strange, it should be done automatically by home assistant.

**Comment by DTTerastar** [comment]
Yep.  My test box w/ no autoupdate stayed up for hours, once deployed all my nodes are up for 30 seconds.  Sorry for the lack of quality as of late.   I will work on it again today and try and figure it out.  I just got my debugger yesterday so it should help.  The only issue is my debugger is a WROVER and WROVER aren't crashing!!!

**Reply by tasi333** [reply]
Thanks @DTTerastar, when do you think we'll get an update to resolve this issue.  

I've got 7 VROOM devices, don't want to downgrade all of them, don't mind waiting for an auto update.
